doc/api/domain.markdown

```diff
@@ -2,304 +2,265 @@
-perhaps we would like to have a separate domain to use for each request.
-
-That is possible via explicit binding.
+There are times when implicit binding to the current `Domain` is not
+appropriate and a particular `EventEmitter` should be bound to a different
+`Domain` object. Doing so can be accomplished via explicit binding using the
+`domain.add()` method.
 
 For example:
 
-```
-// create a top-level domain for the server
-const domain = require('domain');
-const http = require('http');
-const serverDomain = domain.create();
-
-serverDomain.run(() => {
-  // server is created in the scope of serverDomain
-  http.createServer((req, res) => {
-    // req and res are also created in the scope of serverDomain
-    // however, we'd prefer to have a separate domain for each request.
-    // create it first thing, and add req and res to it.
-    var reqd = domain.create();
-    reqd.add(req);
-    reqd.add(res);
-    reqd.on('error', (er) => {
-      console.error('Error', er, req.url);
-      try {
-        res.writeHead(500);
-        res.end('Error occurred, sorry.');
-      } catch (er) {
-        console.error('Error sending 500', er, req.url);
-      }
+    // Create a top-level domain for the server
+    const domain = require('domain');
+    const http = require('http');
+    const serverDomain = domain.create();
+
+    serverDomain.run(() => {
+      // Server is implicitly bound to serverDomain
+      http.createServer((req, res) => {
+        // req and res are also implicitly bound to serverDomain
+        // The preference, however, is to have a separate domain
+        // for each request. First, create the new Domain, then
+        // explicitly add req and res to it.
+        var reqd = domain.create();
+        reqd.add(req);
+        reqd.add(res);
+        reqd.on('error', (er) => {
+          console.error('Error', er, req.url);
+          try {
+            res.writeHead(500);
+            res.end('Error occurred, sorry.');
+          } catch (er) {
+            console.error('Error sending 500', er, req.url);
+          }
+        });
+      }).listen(1337);
     });
-  }).listen(1337);
-});
-```
 
 ## domain.create()
 
 * return: {Domain}
 
-Returns a new Domain object.
+Creates and returns a new `Domain` object. Instances of the `Domain` object are
+not to be created using the `new` keyword.
+
+    const domain = require('domain');
+    const myDomain = domain.create();
 
 ## Class: Domain
 
-The Domain class encapsulates the functionality of routing errors and
-uncaught exceptions to the active Domain object.
+The `Domain` class is an [`EventEmitter`][] that encapsulates the functionality
+of routing errors and uncaught exceptions to the active Domain object.
 
-Domain is a child class of [`EventEmitter`][].  To handle the errors that it
-catches, listen to its `'error'` event.
+When the `Domain` class intercepts an error, it will trigger the `'error'`
+event and pass the caught error to the event handler.
```

Comment:
I would suggest not using terminology such as "caught" that is specific to exceptions/errors when describing domains if possible. Maybe "intercepted" would be less confusing than "caught"?